src/styles/global.css

```
@@ -14,326 +14,339 @@
+    /* Link colour for prose (violet-400 meets 5.1:1 on #0a0a0a) */
+    --colour-link-text: #a78bfa;
+
+    /* Semantic Colours */
+    --colour-success: var(--colour-p3-emerald);
+    --colour-warning: var(--colour-p4-amber);
+    --colour-error: #ef4444;
+    --colour-info: var(--colour-p5-sky);
+
+    /* Typography */
+    --font-sans:
+        "Inter", ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont,
```

Stylelint (17.3.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)